references.json

```diff
@@ -58,6 +58,12 @@
     "authors": ["Mike West"],
     "publisher": "W3C"
   },
+  "CLIENT-HINTS" : {
+    "href" : "http://httpwg.org/http-extensions/client-hints.html",
```

Comment:
Yeah, I guess that is a better URL. I'll change it. As a heads up, the tools.ietf.org version is slightly behind, but we'll be publishing a new draft soon.